src/res/translation/translation_sv_SE.ts

```diff
@@ -119,7 +119,7 @@
     <message>
         <location filename="../../aboutdlgbase.ui" line="72"/>
         <source>Copyright (C) 2005-2021 The Jamulus Development Team</source>
-        <translation type="unfinished"></translation>
+        <translation>Upphovsrätt (C) 2005-2021 The Jamulus Development Team</translation>
```

Comment:
There's still an open discussion if this should be translated: https://github.com/jamulussoftware/jamulus/pull/1097#discussion_r581265793

Reply:
It's a correct translation but let's see what's the decision will be.

Reply:
I think we can just leave it up to the translators. So you can decide what you want to do.

Reply:
I think we've agreed to leave it to translators, I'm going to merge.